chirp/drivers/vgc.py

```diff
@@ -345,7 +345,7 @@
                 chirp_common.PowerLevel("High", watts=50)]
 
 # B-TECH UV-50X3 id string
-UV50X3_id = "VGC6600MD"
+UV50X3_id = b"VGC6600MD"
```

Comment:
I don't think this is right. If this radio doesn't use byte-native serial and mmap, then it should be using the string version when it inspects the stream, right?